### ISSUE TITLE
Peer-to-peer code review

### ISSUE THREAD

**Issue by KanzaTahreem** [KanzaTahreem/Portfolio-Website#10]
You can leave your feedback here 💡
Happy Coding!💻

**Comment by JustoRago** [comment]
Seeing as how I worked on this I don't exactly have any big suggestions. That said, it did bother me at the time that we ended up making the supporting-text element in the headline section an h2 element to make up for the one we didn't have. 

I can't help but feel that it'd be much better from a structural point of view to have all then project cards in a section and have a title for that section which is the missing h2 element. 

Of course Template 1 possesses no such element, but to be honest I always felt that was a lacking in the template from a structural point of view. This is really more a matter for design than code though.
